packages/modules/devices/solaredge/solaredge/config.py

```diff
@@ -24,8 +24,9 @@ def __init__(self,
 
 
 class SolaredgeBatConfiguration:
-    def __init__(self, modbus_id: int = 1):
+    def __init__(self, modbus_id: int = 1, battery_index: int = 1):
```

Comment:
Bei bestehenden SolarEdge-Speichern fehlt die Einstellung battery_index, diese muss in der Konfiguration noch ergänzt werden. Analog zu hier: https://github.com/openWB/core/blob/c12eae88418ac2d512eff0eb8ec3d2a51e0248a1/packages/helpermodules/update_config.py#L1872
Außerdem in Z. 54 die DATASTORE_VERSION um eins hochzählen und deine Funktion ans Ende der Datei setzen und die Versionszahlen analog zu den anderen Funktionen hochzählen. Beim Starten wird die DATASTORE_VERSION geprüft und wenn diese nicht aktuell ist, die Funktionen ausgeführt und so die Konfiguration geupdatet.

Reply:
Es fehlt noch eine neue Methode in der `update_config.py`, die bei bestehenden Speichern den `battery_index` ergänzt.